app/src/main/java/com/framgia/englishconversation/screen/timeline/TimelinePresenter.java

```diff
@@ -4,29 +4,34 @@
 import com.framgia.englishconversation.data.model.UserModel;
 import com.framgia.englishconversation.data.source.callback.DataCallback;
 import com.framgia.englishconversation.data.source.remote.auth.AuthenicationRepository;
-import com.framgia.englishconversation.data.source.remote.timeline.TimelineDataSource;
 import com.framgia.englishconversation.data.source.remote.timeline.TimelineRepository;
 import com.google.firebase.auth.FirebaseUser;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.annotations.NonNull;
```

Comment:
ERROR: Unused import: io.reactivex.annotations.NonNull. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports